refinery/templates/core/data_set.html

```diff
@@ -87,8 +90,13 @@
 
   <!-- Refinery libraries -->
   <script type="text/javascript">
+    var REFINERY_BASE_URL = document.location.host;
```

Comment:
Is this just being moved from somewhere else? I'm not sure how it's used, but it might be important to get the protocol as well.

Reply:
@mccalluc Yes, the globals are going to being revisited as we speak. Focused on just relocating the provenance within the tab.